docs/api/task-manager/health.asciidoc

```diff
@@ -0,0 +1,133 @@
+[[task-manager-api-health]]
```

Comment:
Can we add this to the REST API list so it shows up in the TOC?

Reply:
++

Reply:
Added :)